plugins/inputs/netdata/README.md

```diff
@@ -0,0 +1,64 @@
+# Netdata Input Plugin
+
+The [Netdata](https://github.com/firehol/netdata) plugin reads all metrics
```

Comment:
Please update link to either https://github.com/netdata/netdata or maybe better https://www.netdata.cloud.

Reply:
Mandatory.